libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/kotlin/file/EnumAsEnumBuilder.kt

```diff
@@ -70,45 +71,46 @@ internal class EnumAsEnumBuilder(
   private fun IrEnum.companionTypeSpec(): TypeSpec {
     return TypeSpec.companionObjectBuilder()
         .addProperty(typePropertySpec())
-        .addFunction(safeValueOfFunSpec())
+        .addProperty(knownEntriesPropertySpec())
         .addFunction(knownValuesFunSpec())
+        .addFunction(safeValueOfFunSpec())
         .build()
   }
 
   private fun IrEnum.knownValuesFunSpec(): FunSpec {
     return FunSpec.builder(Identifier.knownValues)
         .addKdoc("Returns all [%T] known at compile time", selfClassName)
+        .addAnnotation(deprecatedAnnotation("Use knownEntries instead").toBuilder().addMember("replaceWith·=·ReplaceWith(%S)", "this.knownEntries").build())
```

Comment:
I think you don't need the `"this"`?

```suggestion
        .addAnnotation(deprecatedAnnotation("Use knownEntries instead").toBuilder().addMember("replaceWith·=·ReplaceWith(%S)", "knownEntries").build())
```

Reply:
Interesting. TIL!